spec/ims/lti/models/lti_model_spec.rb

```diff
@@ -283,6 +288,34 @@ class TestSubclass < SampleClass
 
       end
 
+      context 'unknown methods' do
```

Comment:
Do we have to use instance_variable_get/set for these tests? Is there a public api we can use so we aren't testing internal implementation?